components/compass-runtime-agent/internal/kyma/service.go

```diff
@@ -412,8 +413,9 @@ func getEventApiType(eventApiSpec *model.EventAPISpec) docstopic.ApiType {
 
 func newResult(application v1alpha1.Application, operation Operation, appError apperrors.AppError) Result {
 	return Result{
-		ApplicationID: application.Name,
-		Operation:     operation,
-		Error:         appError,
+		ApplicationName: application.Name,
+		ApplicationIDs:  application.Spec.CompassMetadata.Authentication.ClientIds,
```

Comment:
We should get the `ApplicationID` from the data we get from the Director. In the future `Spec.CompassMetadata.Authentication.ClientIds` will contain something different than the `ApplicationID`.
This will require a little refactoring as we will need to pass the ID to `newResult` function.

Reply:
Done!